recipes-core/iotedge-cli/iotedge-cli_1.1.3.bb

```diff
@@ -0,0 +1,277 @@
+    crate://crates.io/tokio-reactor/0.1.1 \
+    crate://crates.io/tokio-signal/0.2.5 \
+    crate://crates.io/tokio-sync/0.1.7 \
+    crate://crates.io/tokio-tcp/0.1.0 \
+    crate://crates.io/tokio-threadpool/0.1.16 \
+    crate://crates.io/tokio-timer/0.2.11 \
+    crate://crates.io/tokio-tls/0.2.0 \
+    crate://crates.io/tokio-udp/0.1.0 \
+    crate://crates.io/tokio-uds/0.2.5 \
+    crate://crates.io/tokio/0.1.22 \
+    crate://crates.io/treediff/3.0.1 \
+    crate://crates.io/try-lock/0.2.2 \
+    crate://crates.io/typed-headers/0.1.1 \
+    crate://crates.io/typenum/1.10.0 \
+    crate://crates.io/ucd-util/0.1.1 \
+    crate://crates.io/unicase/2.1.0 \
+    crate://crates.io/unicode-bidi/0.3.4 \
+    crate://crates.io/unicode-normalization/0.1.5 \
+    crate://crates.io/unicode-width/0.1.4 \
+    crate://crates.io/unicode-xid/0.1.0 \
+    crate://crates.io/unicode-xid/0.2.0 \
+    crate://crates.io/unreachable/1.0.0 \
+    crate://crates.io/url/1.7.2 \
+    crate://crates.io/url/2.1.1 \
+    crate://crates.io/url_serde/0.2.0 \
+    crate://crates.io/utf8-ranges/1.0.0 \
+    crate://crates.io/vcpkg/0.2.3 \
+    crate://crates.io/vec_map/0.8.0 \
+    crate://crates.io/version_check/0.1.3 \
+    crate://crates.io/version_check/0.9.1 \
+    crate://crates.io/void/1.0.2 \
+    crate://crates.io/want/0.2.0 \
+    crate://crates.io/widestring/0.3.0 \
+    crate://crates.io/winapi-build/0.1.1 \
+    crate://crates.io/winapi-i686-pc-windows-gnu/0.4.0 \
+    crate://crates.io/winapi-x86_64-pc-windows-gnu/0.4.0 \
+    crate://crates.io/winapi/0.2.8 \
+    crate://crates.io/winapi/0.3.8 \
+    crate://crates.io/wincolor/0.1.6 \
+    crate://crates.io/windows-service/0.1.0 \
+    crate://crates.io/winreg/0.5.1 \
+    crate://crates.io/ws2_32-sys/0.2.1 \
+    crate://crates.io/yaml-rust/0.4.0 \
+    crate://crates.io/zip/0.5.3 \
+    git://github.com/Azure/hyperlocal-windows;protocol=https;nobranch=1;name=hyperlocal-windows;destsuffix=hyperlocal-windows \
+    git://github.com/Azure/mio-uds-windows.git;protocol=https;nobranch=1;name=mio-uds-windows;destsuffix=mio-uds-windows \
+    git://github.com/Azure/tokio-uds-windows.git;protocol=https;nobranch=1;name=tokio-uds-windows;destsuffix=tokio-uds-windows \
+"
+
+SRCREV_FORMAT .= "_hyperlocal-windows"
+SRCREV_hyperlocal-windows = "${AUTOREV}"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/hyperlocal-windows"
+SRCREV_FORMAT .= "_mio-uds-windows"
+SRCREV_mio-uds-windows = "${AUTOREV}"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/mio-uds-windows"
+SRCREV_FORMAT .= "_tokio-uds-windows"
+SRCREV_tokio-uds-windows = "${AUTOREV}"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/tokio-uds-windows"
+
+LIC_FILES_CHKSUM=" \
+file://../LICENSE;md5=0f7e3b1308cb5c00b372a6e78835732d \
+file://../THIRDPARTYNOTICES;md5=766e8bf0f17552c5c4cea26775a5b296 \
+"
+
+SUMMARY = "The iotedge tool is used to manage the IoT Edge runtime."
+HOMEPAGE = "https://github.com/Azure/iotedge"
+LICENSE = "MIT"
+
+include iotedge-cli-${PV}.inc
```

Comment:
Is the yocto build machine has no access to the internet?  Otherwise, we might as well have it fetch the value of https://github.com/Azure/iotedge/blob/release/1.1/edgelet/version.txt on the fly. One less file for us to maintain.

Reply:
We have a near term goal to improve automation and reduce manual effort.  I created a PBI and added this and Bear's checksum comment to this.